addons/l10n_ec/demo/account_demo.py

```diff
@@ -15,7 +15,7 @@ def _get_demo_data_move(self):
         cid = self.env.company.id
         model, data = super()._get_demo_data_move()
         if self.env.company.account_fiscal_country_id.code == 'EC':
-            document_type = ref('l10n_ec.ec_dt_18', False) and ref('l10n_ec.ec_dt_18').id or False
+            document_type = ref('l10n_ec.ec_dt_18', False) and ref('l10n_ec.ec_dt_18').id or False  # TODO 18
```

Comment:
=> 01

Reply:
I think we should just use code 01 either for the sale invoice or purchase invoice. The MVP focus is electronic invoicing, not ATS. And even for the ATS, it will be easier to do the mapping just for the ATS report than doing it everywhere else (same is true for reimbursements)

Reply:
@odoostan I think this comment would be closed now as you have already changed the code to 01. Would you close it please?